test/osutils.jl

```diff
@@ -0,0 +1,8 @@
+@test (@static true ? 1 : 2) == 1
```

Comment:
The other alternative is keeping an osutils.jl test file but making it worth the cost of spawning another test process by moving a few of the existing relevant os tests from other files into it.